tooling/nargo_cli/src/cli/mod.rs

```diff
@@ -112,3 +112,20 @@ pub(crate) fn start_cli() -> eyre::Result<()> {
     println!("{markdown}");
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use clap::Parser;
+    /// Test that parsing an invalid option doesn't
```

Comment:
Unfinished sentence :-)

We could either finish it or remove this comment, your choice.

Reply:
Thanks, removed. I wanted to say "doesn't cause a panic" but then realised that it never did, froze mid sentence.